### PR TITLE
Test and verify using Git Actions

### LINKED ISSUE #3825: Travis-ci.org Shutdown - Migration to Travis-ci.com
**Which component are you using?**:
N/A

**What version of the component are you using?**:

N/A

**What k8s version are you using (`kubectl version`)?**:

N/A

**What environment is this in?**:

N/A

**What did you expect to happen?**:

Travis CI builds should be processed through https://travis-ci.com

**What happened instead?**:

The deprecated https://travis-ci.org is still used, potentially breaking our builds once the old domain is switched off.

**How to reproduce it (as minimally and precisely as possible)**:

Visit a travis build from a PR to this repository:
https://travis-ci.org/github/kubernetes/autoscaler/builds/752730430

**Anything else we need to know?**:

Travis' announcement of the intended shutdown of travis-ci.org:
https://mailchi.mp/3d439eeb1098/travis-ciorg-is-moving-to-travis-cicom

Travis' instructions for moving to travis-ci.com:
https://docs.travis-ci.com/user/migrate/open-source-repository-migration

### PR DESCRIPTION
**Pull request by mwielgus**
As Travis or is about to shut down, we are migrating to some alternative. 
Fixes #3825

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  GOPATH: ${{ github.workspace }}/go
+
+jobs:
+  test-and-verify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
```
Comment (MaciekPytel) [inline .github/workflows/ci.yaml:17]: I think we're at 1.15 now (at least in CA).

**Review by MaciekPytel — COMMENTED** [review]
Are you planning to remove travis config in a follow-up PR? If so this looks good to me, though I have little expertise in github workflows.

@gjtempleton do you want to also review? I think you're more qualified than I am.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (mwielgus) [reply .github/workflows/ci.yaml:17]: Travis is at 1.14

https://github.com/kubernetes/autoscaler/blob/master/.travis.yml#L9

Let's go with 1.14. Then we can update to 1.15.

Comment (MaciekPytel) [reply .github/workflows/ci.yaml:17]: I think travis being on 1.14 is a bug (we must have forgotten - again - to bump it when bumping go version). https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/go.mod#L9

Comment (mwielgus) [reply .github/workflows/ci.yaml]: Ok, whatever. Flipped to 1.15.

**Pull request merged**